src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo
+}
+
+export const TodoInfo: FC<Props> = ({ todo: { completed, title } }) => (
```

Comment:
Better destructure it in the component because nested destructuring is not readable